<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profits</class>
 <widget class="QDialog" name="Profits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1051</width>
     <height>791</height>
    </rect>
   </property>
   <property name="title">
    <string>Profits</string>
   </property>
   <widget class="QGroupBox" name="filterTwoDatesGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1021</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter By Two Dates</string>
    </property>
    <widget class="QGroupBox" name="filterTwoDatesProfitsAllShopsGroupBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Total Profits All shops</string>
     </property>
     <widget class="QLabel" name="filterTwoDatesProfitsLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>£0.00</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="filterTwoDatesShopsGroupBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Total Profits Per Shop</string>
     </property>
     <widget class="QLabel" name="filterTwoDatesProfitsShopsLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>£0.00</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>31</y>
        <width>199</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="filterTwoDatesShopsLabel">
         <property name="text">
          <string>Current Shop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filterTwoDatesShopsComboBox"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="filterTwoDatesRefreshButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>60</y>
       <width>88</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>208</width>
       <height>39</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="dateOneLabel">
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dateOneLineEdit">
        <property name="text">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>208</width>
       <height>39</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="dateTwoLabel">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dateTwoLineEdit">
        <property name="text">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1021</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter All time By Shop</string>
    </property>
    <widget class="QLabel" name="filterAllTimeShopsProfitsLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>£0.00</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>199</width>
       <height>38</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="filterAllTimeShopsLabel">
        <property name="text">
         <string>Current Shop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterAllTimeShopsComboBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>1021</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>View Graphs</string>
    </property>
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>991</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>439</y>
       <width>518</width>
       <height>38</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="startDateLabel">
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startDateLineEdit">
        <property name="text">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="EndDateLabel">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endDateLineEdit">
        <property name="text">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="graphsRefreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="allShopsCheckBox">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>430</y>
       <width>88</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>All shops</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="shop2CheckBox">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>460</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Shop 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="shop1CheckBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>430</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Shop 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="shop3CheckBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>460</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Shop 3</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
